python/cudf/cudf/_lib/io/utils.pyx

```diff
@@ -67,3 +72,61 @@ cdef cppclass iobase_data_sink(data_sink):
 
     size_t bytes_written() with gil:
         return buf.tell()
+
+
+cdef class HostBuffer:
```

Comment:
Do we want to consider exposing buffer protocol for this class?

Reply:
Should we use this name here? Thought this was going to be reserved for the pinned memory allocation made by RMM (once that arrives). ( https://github.com/rapidsai/rmm/issues/260 )

Reply:
Ah good call, yes we shouldn't use `HostBuffer` here.